app/views/items/_navbar.html.slim

```diff
@@ -1,8 +1,11 @@
 nav.navbar.navbar-expand-lg.bg-light.justify-content-between.px-4
   .navbar-nav.ml-auto
     = link_to image_tag("logo.png"), root_path, class: "nav-link"
-    .nav-link.organization-name
-      = current_user&.organization&.name
+    .company-name
+      - if current_user&.organization && current_user&.director?
```

Comment:
Don't you want to move this condition to policy method and write here smth like `- if allowed_to(:show_organization)`?

Reply:
I think thats change not make this fragment more readable, this will be change later